### PR TITLE
Removed short content from blog cards

### PR DESCRIPTION
**Pull request by A7abhilash**
## What is the change?
removed the short content visible on blog cards in home(/), category(/category), dashboard(/dashboard) and author(/author/:authorId) UI.
Correspondingly added posts partial as we are displaying the posts(blogcards) in same way in the above mentioned routes.

## Related issue?
close: #447 
Old pr: #462 

## How was it tested?
Tested the functioning and responsiveness thoroughly in localhost.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

### CODE REVIEW

File: views/partials/posts.ejs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                    <%= post.author.userName %> 
+                </a>
+            </div><hr>
+            <h1 style="font-size: 2.4rem;font-weight: bold;"> <%= post.blogTitle %> </h1>
+            <p class="post-date-font">Posted on <%= moment(new Date(post.timestamps)).format("dddd, MMMM Do YYYY, h:mm:ss a") %> </p>
+            <% if (post.category) { %>
+                <div class="badge badge-pill badge-danger py-1 px-2">
+                    <%= post.category %> 
+                </div>
+            <% } %>
```
Comment (Pranav016) [inline views/partials/posts.ejs:16]: can we add a sneak peek of the post content like this
```
<div class="post-content-font">
    <%- post.blogContent.substring(0,1000) + ("...") %>
</div>
```

**Review by Pranav016 — CHANGES_REQUESTED** [review]

File: views/partials/posts.ejs (hunk 1 above)
Comment (A7abhilash) [reply views/partials/posts.ejs:16]: If you say so then i can add... 
Should i make a separate or for that or make a new one?

Comment (A7abhilash) [reply views/partials/posts.ejs:16]: Umm.. I think lets move on with current feature... We can discuss adding the sneak peak in discord and come to a conclusion...

Comment (Pranav016) [reply views/partials/posts.ejs:16]: I think sneak peek in the post was already there before this PR, I mean we can still discuss if you want to..

Comment (A7abhilash) [reply views/partials/posts.ejs:16]: This PR is for removing that sneak peek(short content) from the cards..

Comment (Pranav016) [reply views/partials/posts.ejs:16]: I wanted to have the sneak peek, but I think you have raised a new issue to add that ?
So think I should merge this one..

**Comment by Pranav016** [comment]
@A7abhilash resolve conflicts carefully, I think it is ready to merge

**Comment by A7abhilash** [comment]
> @A7abhilash resolve conflicts carefully, I think it is ready to merge

Yeah yeah🙌
I dont think there are conflicts... The master branch has the div of displaying posts but i had now kept it in partials right(you can also check this once)... So i think u can go on to merge

**Comment by Pranav016** [comment]
> > @A7abhilash resolve conflicts carefully, I think it is ready to merge
> 
> Yeah yeah🙌
> I dont think there are conflicts... The master branch has the div of displaying posts but i had now kept it in partials right(you can also check this once)... So i think u can go on to merge

I know it's just a partial added for the post but the merge button doesn't appear untill you resolve them..

**Comment by A7abhilash** [comment]
> > > @A7abhilash resolve conflicts carefully, I think it is ready to merge
> > 
> > 
> > Yeah yeah🙌
> > I dont think there are conflicts... The master branch has the div of displaying posts but i had now kept it in partials right(you can also check this once)... So i think u can go on to merge
> 
> I know it's just a partial added for the post but the merge button doesn't appear untill you resolve them..

Umm... Then shall i make a new pr by resolving the conflict and then later in next pr ill add the partials on home ui also?

**Comment by Pranav016** [comment]
> > > > @A7abhilash resolve conflicts carefully, I think it is ready to merge
> > > 
> > > 
> > > Yeah yeah🙌
> > > I dont think there are conflicts... The master branch has the div of displaying posts but i had now kept it in partials right(you can also check this once)... So i think u can go on to merge
> > 
> > 
> > I know it's just a partial added for the post but the merge button doesn't appear untill you resolve them..
> 
> Umm... Then shall i make a new pr by resolving the conflict and then later in next pr ill add the partials on home ui also?

No, resolving doesn't require to make a new PR. Just click on `resolve conflicts` button and you can resolve them on github

**Comment by Pranav016** [comment]
resolved them

**Review by Pranav016 — APPROVED** [review]

**Comment by Pranav016** [comment]
/gssoc/

**Comment by Pranav016** [comment]
/level2/

**Pull request merged**